station-flattening-v0/src/main/java/net/modificationstation/stationapi/mixin/flattening/MixinChunk.java

```diff
@@ -101,7 +101,6 @@ private void onChunkCreation(Level level, byte[] tiles, int x, int z, CallbackIn
             BlockStateHolder holder = BlockStateHolder.class.cast(BlockBase.BY_ID[id]);
             section.setBlockState(bx, by & 15, bz, holder.getDefaultState());
         }
-        this.tiles = null;
```

Comment:
Why is this removed?

Reply:
Looks like this field is in use somewhere, if it will be null generation will fall into infinity loop. I actually not sure why as this thing works in old worlds

Reply:
I think that MC calls this field for new worlds somewhere in the player spawn check, and when it can't find spawn position - it starts chunk creation in a loop. As it can't find position - loop will be infinity. Actually, this should be fixed later, this is just "fast" fix that allows worlds to be created